doc/index.rst

```diff
@@ -278,6 +278,11 @@ Special typing primitives
 
       The implementation was changed for compatibility with Python 3.12.
 
+   .. versionchanged:: 4.8.0
+
+      Fixed a bug where passing an ellipsis literal to ``default`` would raise a 
+      :py:exc:`TypeError` on Python versions <3.11.
+
```

Comment:
Over at CPython we usually _wouldn't_ advertise pure bugfixes in the main docs (only in the changelog -- in the main docs we generally pretend the implementation has been perfect all along ;) but I'm not sure what our policy is at `typing_extensions`.

(The line between bugfixes and features is also slightly blurrier in `typing_extensions`, since "backporting a bugfix from CPython" can count as a "`typing_extensions` feature". But the feature we're fixing a bug in here hasn't been implemented in CPython yet, of course.)

Reply:
I'm OK with adding it, but would reword it as "Passing an ellipsis literal (`...`) to `default` now works on Python 3.10 and lower."